packages/client/src/features/shortcuts/shortcuts-manager.ts

```diff
@@ -0,0 +1,58 @@
+/********************************************************************************
```

Comment:
I think we should move this functionality into the default module (i.e. the base package).
Having it as a dedicated feature module means that adopters could choose to remove/not load this module.
This would break all services that inject the shortcut manager